core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.css

```
@@ -0,0 +1,80 @@
+}
+.oppia-thumbnail-uploader {
+  word-break: break-word;
+}
+.oppia-thumbnail-confirm-button {
+  position: absolute;
+  right: -50px;
+  top: 40px;
+}
+.oppia-thumbnail-colour-select {
+  display: grid;
+  grid-gap: 5px;
```

Stylelint (17.14.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)